tests/BenchmarkDotNet.Tests/Running/JobRuntimePropertiesComparerTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Linq;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Attributes.Jobs;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Running;
+using Xunit;
+
+namespace BenchmarkDotNet.Tests.Running
+{
+    public class JobRuntimePropertiesComparerTests
```

Comment:
It's cool that now we are using fast unit tests for such cases instead of slow integration tests!